examples/graphene_apps/python_workloads/fibonacci/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM avalon-python-worker-dev
+
+# Ignore timezone prompt in apt
+#ENV DEBIAN_FRONTEND=noninteractive
+
+# Add necessary packages
+RUN apt-get update \
+ && apt-get install -y -q \
+    ca-certificates \
+    pkg-config \
+    python3-dev \
+    python3-pip \
+    python3-toml \
+    gcc \
+ && python3 -m pip install pyzmq pycryptodomex ecdsa\
+ && apt-get clean 
+
+# Make Python3 default
+RUN ln -sf /usr/bin/python3 /usr/bin/python
+
+COPY . /home/python_workloads/fibonacci
+
+WORKDIR /home/python_workloads/fibonacci
+
+# Build fibonacci python workload module and install
+RUN make && make install
+
+# GSC (Graphene Shielded Container) docker image build depends on child
+# application manifest files.
+# Python worker makes use of Python Cryptodomex which indirectly depends on
+# multiple applications and manifest files has to be listed in GSC build.
+# gcc depends on collect2 and GSC build exits with below message.
+# "Command 'which collect2' returned non-zero exit status 1".
+# Add a temporary workaround to add collect2 to path.
+ENV PATH=$PATH:/usr/lib/gcc/x86_64-linux-gnu/7.5.0/
```

Comment:
Do we need to add this PATH here. It should be in avalon worker Dockerfile

Reply:
Not needed as its already set in python worker docker image. Will remove.